localstack/services/dynamodb/dynamodb_listener.py

```diff
@@ -159,10 +167,9 @@ def forward_request(self, method, path, data, headers):
             if not self.table_exists(ddb_client, data["TableName"]):
                 return get_table_not_found_error()
 
-        elif action == "DeleteTable":
-            # Check if table exists, to avoid error log output from DynamoDBLocal
-            if not self.table_exists(ddb_client, data["TableName"]):
```

Comment:
I'd prefer if we can keep the previous format of the `if`-checks here - otherwise we may end up doing multiple unnecessary comparisons (or even run into undesired behavior) with the remaining `elif` branches below... (e.g., if `action` is `"DeleteTable"`, and the table _does_ exist.